docs/how-to-profile-a-local-unity-build-remotely.md

```diff
@@ -1,46 +1,18 @@
-3. After that, run the bundling process in `unity-renderer` by executing `npm run build` in the `./browser-interface` folder.
-4. When the building process finishes, move all the "files" files that will end up inside `browser-interface/dist` into `kernel/static/unity-renderer`.
-
-Just to be clear, the generated files should end up in the following path
-
-```
-kernel
- /- static
-    /- unity-renderer
-      /- index.js
-      /- unity.data.unityweb
-      /- unity.wasm.code.unityweb
-      /- unity.wasm.framework.unityweb
-      /- ...
-```
-
-#### Option 2
-
-1. Run `make watch` in a terminal at `kernel/`. Wait for it to finish and leave it watching for changes. **This step must be done before replacing the build files or they may be replaced again with the latest published build files when the `make watch` process runs**
-2. Build unity WASM with its name as `unity` into the folder `kernel/static`. **It's very important that the folder/build name is `unity`**.
-3. Testing how your new build performs:
-
-- Open **[http://localhost:3000/?DEBUG_MODE&LOCAL_COMMS&position=-100,100](http://localhost:3000/?DEBUG_MODE&LOCAL_COMMS&position=-100,100)** to go to an area with a high density of test parcels.
-- Open **[http://localhost:3000/?DEBUG_MODE&LOCAL_COMMS&ENV=org&position=10,0](http://localhost:3000/?DEBUG_MODE&LOCAL_COMMS&ENV=org&position=10,0)** to open an area with real-life deployments (but without communicating with other users).
-- Open **[http://localhost:3000/?ENV=org&position=0,90](http://localhost:3000/?ENV=org&position=0,90)** to open the explorer near the Decentraland Museum
-
 ### Profiling a build with Unity Editor
 
-1. In Unity Editor open the profiler at window -> analysis -> profiler
-2. Open the build settings and toggle on "Development Build", "Autoconnect Profiler" and **make sure Deep Profiling is toggled off** before starting the build
-3. Follow the steps at this readme's "Making a manual build" section
+1. Follow the [local build steps](https://github.com/decentraland/unity-renderer#debug-with-browsers--local-unity-build) but for step 4 apply the following settings before building:
+* Open the Player Settings and go to the **compression** setting and set it to **Disabled**
+* Open the build settings and toggle on "Development Build", "Autoconnect Profiler" and **make sure Deep Profiling is toggled off**
+2. In Unity Editor open the profiler at window -> analysis -> profiler
+3. Since Unity2020, the WebGL development build doesn't produce the same 3 `.unityweb` files, instead there is a `.wasm` file and two `.js` files. Move those files into the corresponding kernel folder (right next to the production `.unityweb` files) and then update [these 3 lines](https://github.com/decentraland/explorer/blob/c3df43602c99d90142d7bd0841d378097a6a9ed6/kernel/packages/unity-interface/loader.ts#L52) to point to your newly created build files
```

Comment:
is it OK that url is pointing to a branch instead of master?
although, replacing that url to master's means that the line number might change... 🤔 
could we add a code snippet here just to make it absolutely clear?